doc/sphinx/language/gallina-extensions.rst

```diff
@@ -2260,3 +2260,52 @@ expression as described in :ref:`ltac`.
 This construction is useful when one wants to define complicated terms
 using highly automated tactics without resorting to writing the proof-term
 by means of the interactive proof engine.
+
+.. _primitive-integers:
+
+Primitive Integers
+--------------------------------
+
+The language of terms features 63-bit machine integers as values. The type of
+such a value is *axiomatized*; it is declared through the following sentence
+(excerpt from the :g:`Int63` module):
+
+.. coqdoc::
+
+   Primitive int := #int63_type.
```

Comment:
I wonder about the choice to name this type `int`. It doesn't convey the 63-bit nature. If it means that the module should not be imported (and used qualified), what about naming it `t` (as in standard OCaml conventions)?

Reply:
The fact that there are 63 bits is an accident. Should this be reflected in the name of the type of “primitive machine integers”?

Reply:
I don't really get what you are proposing. But my idea was indeed that this should be reflected, either as `Int63.t` or as `Int63.int63`, whichever you prefer. Anyway, this is outside of the scope of this PR.